static/add.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function() {
+  $("input").eq(1).on("click", function(e) {
```

Comment:
Hmmmmm.  This is a bit odd.  Using the `.eq` filter to reduce the set of matched elements to the one at a specific index is brittle, since it assumes that the item you meant is always going to be at the same index.  It might be better to pursue a strategy where you use `id` or `class` attributes of the HTML input element to get the one you want.